crates/cli/src/cli.rs

```diff
@@ -70,7 +70,7 @@ fn main() -> anyhow::Result<()> {
         }
     }
 
-    let re = Regex::new(r"^0x[a-fA-F0-9]{64}$").unwrap();
+    let re = Regex::new(r"^0x[a-fA-F0-9]").unwrap();
```

Comment:
We still need to ensure that class hash are of the correct max length of 66 (which this regex does, by doing 0x+64 chars).  It'll be preferable if we can still have this assertion whilst allowing those that don't have prefixed zeros to work.

Additionally, it'll be great if we can add some tests for this.